solarforecastarbiter/reports/figures/plotly_figures.py

```diff
@@ -331,8 +352,12 @@ def _plot_fx_timeseries(fig, timeseries_value_df, timeseries_meta_df, axis):
     # construct graph objects in random hash order. collect them in a list
     # along with the pair index. Then add traces in order of pair index.
     gos = []
+
     # construct graph objects in random hash order
-    for fx_hash in np.unique(timeseries_meta_df['forecast_hash']):
+    non_distribution_indices = timeseries_meta_df['distribution'].isna()
+    non_distribution_meta = timeseries_meta_df[non_distribution_indices]
+    distribution_meta = timeseries_meta_df[~non_distribution_indices]
+    for fx_hash in np.unique(non_distribution_meta['forecast_hash']):
```

Comment:
perhaps this logic belongs in `timeseries` and then we make a new function `_plot_fx_distribution_timeseries` for the code added below

Reply:
Alright, so I've added the `_plot_fx_distribution_timeseries` function here. I was bashing my head against a bug in the report I included, but it seems like a possible bug unrelated to the plotting code. 
For some reason, the distribution contains values for the 10th percentile, but the `ProcessedForecastObservation` included in the report with the same 10th percentile `ProbabilisticForecastConstantValue` has an empty `forecast_values` attribute. There is a metrics error stating that the  constant value has no data, so I'll have to dig to figure out what the actual issue is, but I don't believe it's related to this PR.

The plots seemed to be working when the forecast plotting functions were merged together because the distribution's data was being used.

Reply:
This issue was due to a reference forecast missing data, and with it removed, everything functions as expected.